src/libs/actions/Policy.js

```diff
@@ -112,6 +112,16 @@ function getPolicyList() {
         });
 }
 
+/**
+ * Get the number of Workspaces administered by the user
+ *
+ * @returns {Number}
+ */
+function getWorkspaceCount() {
+    return _.size(_.chain(allPolicies)
+        .filter(policy => policy && policy.type === CONST.POLICY.TYPE.FREE && policy.role === CONST.POLICY.ROLE.ADMIN).value());
```

Comment:
I think the `_.chain` and `.value` here are unnecessary, we are applying only one filter.

Reply:
got it! 👍